docs/azure/sdk/dependency-injection.md

```diff
@@ -25,34 +35,32 @@ dotnet add package Azure.Security.KeyVault.Secrets
 dotnet add package Azure.Storage.Blobs
 ```
```

Comment:
Not sure if it matters, but they are now told to install Microsoft.Extensions.Azure and Azure.Identity twice - should this second block just be the two service packages?

Reply:
That's a fair point. I'll update it.